src/js/media-controller.js

```diff
@@ -97,7 +97,7 @@ class MediaController extends MediaContainer {
           mediaUIEventHandlers['MEDIA_SEEK_TO_LIVE_REQUEST'](e, media);
         }
 
-        this.media.play();
+        this.media.play().catch(() => {});
```

Comment:
suggestion(non-blocking): while you're in here, you might want to use `this.media?.play()` etc. or `this.media.play?.()` (since we haven't defined a "minimally valid slotted media element" yet) to catch those edge cases as well